src/index.html

```diff
@@ -4,13 +4,52 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <div class="form">
+      <form
+      action="http://www.google.com"
+      method="post"
+      data-qa="big"
+      class="form-search"
+      >
+        <label for="search-bar-one" >
+
+          <input
+          type="search"
+          name="search-bar-one"
+          id="searchBarOne"
+          class="search-bar search1"
+          placeholder="Try &quot;Los Angeles&quot;"
+          onfocus="this.placeholder='Los Ang'"
+          data-qa="keypress"
```

Comment:
In focus state placeholder is not fully visible it cuts short, refer the screenshot
<img width="363" alt="image" src="https://user-images.githubusercontent.com/100297341/196447883-bef8f6be-674b-45c5-b718-fd3fee191399.png">

Reply:
That is how it is shown on the figma drawing. Shall I change it back to the full phrase?